bash_completion.bash

```diff
@@ -250,4 +255,4 @@ _mvn()
 }
 
 complete -o default -F _mvn -o nospace mvn
-complete -o default -F _mvn -o nospace mvnDebug
+complete -o default -F _mvn -o nospace mvnDebug
```

Comment:
why the change here?

Reply:
I don't see the difference either.  maybe and add space